src/main/java/com/ubc/cpsc304/hotelify/controller/dto/ReviewRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.ubc.cpsc304.hotelify.controller.dto;
+
+import lombok.Data;
+
+@Data
+public class ReviewRequestDto {
+    private Long review_id;
```

Comment:
I don't think review_id should be included in the reviewRequestDto since if we haven't created it, how do we know the id?

Reply:
I'm a bit confused with requestDTO & responseDTO, in this cirsumstances, do you think requestDTO should include hotel_id and username, and responseDTO include comment and rating?

Reply:
requestDto should include hotel_id and username, I think, because its a user types a review for a hotel, front-end should be having these info. for response, we would want to return as much as we want without letting the front-end knowing too much. in this case, we can return id, comment and rating. 其实我们不需要写requestDTO & responseDTO,我早上和 @suiyoubi 讨论了下，咱们project其实根本没必要写 response dto，直接 return entity 其实就行。 但是我们需要request dto,直接叫dto就行